server/public/index.html

```diff
@@ -5,6 +5,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Final Project</title>
   <link rel="icon" href="/favicon.ico">
+  <script src="https://kit.fontawesome.com/70432710ae.js" crossorigin="anonymous"></script>
```

Comment:
I'd suggest the font-awesome script wait until after all the `"preconnect"`s so it doesn't slow down the rest of the page loading so much.